docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,607 @@
-
-```code
-$ curl https://${PROXY?}/webapi/ping | jq .proxy.ssh.public_addr
-"mytenant.teleport.sh:443"
-```
-
-</TabItem>
-
-</Tabs>
-
-#### `spec.role_map`
-
-When a leaf cluster establishes trust with a root cluster, it needs a way to
-configure access from users in the root cluster. Teleport enables you to limit
-access to Trusted Clusters by mapping Teleport roles to cluster labels.
-
-When creating a `trusted_cluster` resource, the administrator of the leaf
-cluster must define how roles from the root cluster map to roles on the leaf
-cluster.
-
-`trusted_cluster.yaml` uses the following configuration:
-
-```yaml
-  role_map:
-    - remote: "access"
-      local: ["visitor"]
-```
-
-Here, if a user has the `access` role on the root cluster, the leaf cluster will grant
-them the `visitor` role when they attempt to log in to a Node.
-
-If your user on the root cluster has the `access` role, leave this as it is. If
-not, change `access` to one of your user's roles.
+   ```yaml
+   kind: trusted_cluster
+   version: v2
+   metadata:
+     name: rootcluster.example.com
+   spec:
+     enabled: true
+     token: (=presets.tokens.first=)
+     tunnel_addr: rootcluster.example.com:443
+     web_proxy_addr: rootcluster.example.com:443
+     role_map:
+       - remote: "access"
+         local: ["visitor"]
+   ```
+
+   In this file:
+
+   - Set `metadata.name` to the name of your root cluster.
+   - Set `spec.token` to the invitation token you generated previously.
+   - Set `spec.tunnel_addr` to the reverse tunnel address of the Teleport Proxy Service in the root cluster.
+   - Set `spec.web_proxy_addr` to the address of the Teleport Proxy Service on the root cluster. 
+   - Set `spec.role_map` to map Teleport roles from the root cluster to roles in the leaf cluster.
+
+   If you aren't sure of what values to set for the `tunnel_addr` or `web_proxy_addr`,
+   see [Commands to look up cluster addresses](./trusted-cluster-address-lookup.mdx).
+
+   You use the `role_map` setting in the `trusted_cluster` resource configuration to define how 
+   roles from the root cluster map to roles on the leaf cluster. In this example, users assigned 
+   the `access` role for the root cluster are granted the `visitor` role when they attempt to log 
+   in to a server in the leaf cluster. This role mapping enables you to restrict access to resources
+   in the leaf cluster.
+   
+   If your Teleport user is assigned the `access` role on the root cluster, you can use this role 
+   mapping to test access to the server in the leaf cluster. If your Teleport user isn't assigned 
+   the `access` role, change `access` in the `role_map` to one of your user's roles.
+
+   Role mapping can be quite powerful for managing access in trusted clusters. For more information
+   about how to use role mapping to restrict access to leaf clusters and examples of the accepted
+   syntax, see [Role mapping  in trusted clusters](#role-mapping-in-trusted-clusters).
```

Comment:
I think this paragraph and internal link add unnecessary branching. Since this is a previous section in the same page, I think a user will had the opportunity to read it already.

Reply:
This link is to a topic further down the page. The headings are too similar.